### PR TITLE
CAPWAP upstream release 1.9.1

### PR DESCRIPTION
**Pull request by platinumthinker**

### CODE REVIEW

**Comment by tlnd** [comment]
I'm wondering why this PR was not pushed to docker hub. According to the travis logs, it was pushed, however, I don't see recent [tags in docker hub](https://hub.docker.com/r/ergw/ergw-capwap-node/tags/).
@RoadRunnr any idea?

**Comment by tlnd** [comment]
#5 updates the docker password which causes images to be pushed again.

File: rebar.config [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {erl_opts, [debug_info]}.
 {deps, [
-	{ergw, ".*", {git, "git://github.com/travelping/capwap", {tag, "1.8.0"}}},
+	{capwap, ".*", {git, "git://github.com/travelping/capwap", "master"}},
```
Comment (RoadRunnr) [inline rebar.config]: this should be a tag off the capwap project.

Comment (platinumthinker) [reply rebar.config]: yes, but tag 1.9 does not have my changes..

Comment (RoadRunnr) [reply rebar.config]: then you need to prepare a 1.9.1 release of capwap with you changes.

**Comment by RoadRunnr** [comment]
should the version of the ergw-capwap-node match the version of capwap?

**Pull request closed without merging**